codespell_lib/data/dictionary.txt

```diff
@@ -21157,14 +21157,26 @@ lables->labels
 labling->labeling, labelling,
 labouriously->laboriously
 labratory->laboratory
+labrynth->labyrinth
+labrynths->labyrinths
+lacker->lacquer
+lackered->lacquered
+lackeres->lacquers
+lackering->lacquering
+lackers->lacquers
+lackrimose->lachrymose
+lackrimosity->lachrymosity
+lackrimosly->lachrymosely
 laer->later, layer,
+laf->laugh
 lagacies->legacies
 lagacy->legacy
 laguage->language
 laguages->languages
 laguague->language
 laguagues->languages
 laiter->later
+lambast->lambaste
```

Comment:
It looks like `lambast`/`lambasts` are accepted variants of `lambaste`/`lambastes`, at least in the UK:
* [Cambridge](https://dictionary.cambridge.org/dictionary/english/lambaste)
* [Oxford Learner's](https://www.oxfordlearnersdictionaries.com/definition/english/lambast)
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/lambast)
* [Collins](https://www.collinsdictionary.com/dictionary/english/lambast)

Move to GB_US?

Reply:
> Move to GB_US?

Yup. I also found this reference: https://grammarist.com/spelling/lambast-lambaste/

Reply:
Another one of these:

```
E   error 'lambast' should be in aspell dictionaries (en_GB) for dictionary /home/skangas/wip/codespell/codespell_lib/tests/../data/dictionary_en-GB_to_en-US.txt
E   error 'lambasts' should be in aspell dictionaries (en_GB) for dictionary /home/skangas/wip/codespell/codespell_lib/tests/../data/dictionary_en-GB_to_en-US.txt
```

So I've added it to the list in #2657 instead, and removed it from this PR.